.goreleaser.yml

```diff
@@ -1,5 +1,5 @@
 env:
-  - OPTIMUS_RELEASE_URL=https://github.com/odpf/optimus/releases/download/v0.2.0-alpha/optimus_0.2.0-alpha_linux_x86_64.tar.gz
+  - OPTIMUS_RELEASE_URL=https://github.com/odpf/optimus/releases/download/v0.2.6-rc.1/optimus_0.2.6-rc.1_linux_x86_64.tar.gz
```

Comment:
lets update with the latest release before merging

Reply:
updated. however, we are just using the one in the main branch (of this repository)